src/UI/Component/Table/PresentationRow.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/* Copyright (c) 2017 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Table;
+
+use ILIAS\UI\Component\Triggerable;
+
+/**
+ * This describes a Row used in Presentation Table.
+ * A row consists (potentially) of title, subtitle, important fields (in the
+ * collapased row) and further fields to be shown in the expanded row.
+ */
+interface PresentationRow extends \ILIAS\UI\Component\Component, Triggerable {
```

Comment:
I currently can't see why we need getters on this (public) interface. We only use it for the construction of a row in the row_mapping, right?